.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+<!--
+    Anything in <!-- and -> are comments and would not appear to us
+-->
+
+<!--
+    We categorize our issues into 2 categories:
+    * Bug: Prefix your issue title with [BUG]
+    * Feature Request: Prefix your issue title with [FEATURE]
+
+    Note you do not need to provide all the fields given but the more you give the better and easier for us to figure
+    out the issue.
```

Comment:
"This template is a suggestion of how to layout your report. Whilst for most cases this template should be followed, feel free to modify it if it fits your problem or feature request more. The important thing is that you give us as much information as possible so that we can solve your issue, or potentially implement the feature with minimal fuss."